includes/class-newspack-popups-inserter.php

```diff
@@ -522,6 +523,37 @@ public static function insert_inline_prompt_in_archive_pages( $post_count ) {
 		}
 	}
 
+	/**
+	 * Modify the view script tag to allow it as an "AMP Plus" script.
+	 *
+	 * @param string $tag HTML of the script tag.
+	 * @param string $handle The script handle.
+	 */
+	public static function mark_view_script_as_amp_plus_allowed( $tag, $handle ) {
+		if ( 'newspack-popups-view' === $handle ) {
+			return str_replace( '<script', '<script data-amp-plus-allowed', $tag );
```

Comment:
Nit: this could be done in `enqueue_scripts` using the script data, [like here](https://github.com/Automattic/newspack-newsletters/blob/2b52e1a0890af8680778c080de61eeffff729cc2/src/blocks/subscribe/index.php#L53). This is processed by Newspack plugin to add the inline attribute.

Reply:
Good call! b4af145 refactors to use `wp_script_add_data` and 7a69c83 makes the script `async`, which doesn't seem to have any negative effect on runtime.